### PR TITLE
Fix details in needle editor

### PR DESCRIPTION
**Pull request by Martchus**
See commit descriptions and #1227

### CODE REVIEW

**Comment by Martchus** [comment]
Not sure why the test fails:

> 500 Internal Response: Cannot read response header: timeout at /home/travis/perl_modules/lib/perl5/Devel/Cover/Report/Codecov.pm line 137.
> Cannot read response header: timeout at /home/travis/perl_modules/lib/perl5/Devel/Cover/Report/Codecov.pm line 137.

File: templates/step/edit.html.ep [hunk 1]
```diff
@@ -182,7 +182,9 @@
                                 <div>
                                     <input type="checkbox" name="tags" id="tag_<%= $tag %>" class="tag_checkbox"
                                             value="<%= $tag %>" <%= 'readonly="readonly" disabled="disabled"' unless (is_operator) %> >
-                                    %= $tag
+                                    <label for="tag_<%= $tag %>">
+                                        %= $tag
+                                    </label>
```
Comment (okurz) [inline templates/step/edit.html.ep]: would you like to make this

```perl
%= label_for "tag_$tag" => $tag
```

Comment (Martchus) [inline templates/step/edit.html.ep]: Changed it

**Comment by Martchus** [comment]
I don't know how the coverage change relates to my changes.

**Comment by aaannz** [comment]
Don't bother with the coverage changes. My understanding is that sometimes travis job is slow triggering a retry in worker registration and/or api calls and sometimes not. Hence change in coverage in unrelated files. Will be fixed as soon as I finish tests for #1215

**Comment by Martchus** [comment]
Ok, good to know.

**Comment by aaannz** [comment]
See https://progress.opensuse.org/issues/17666

**Pull request merged**